ajax/libs/videojs-wavesurfer/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "videojs-wavesurfer",
+  "version": "1.3.2",
+  "description": "video.js plugin that adds a navigable waveform for audio and video files.",
+  "filename": "videojs.wavesurfer.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/collab-project/videojs-wavesurfer.git"
+  },
+  "keywords": [
+    "waveform",
+    "audio",
+    "video",
+    "wavesurfer",
+    "videojs",
+    "videojs-plugin",
+    "player"
+  ],
+  "author": "Collab",
+  "license": "MIT",
+  "homepage": "http://videojs.com/",
```

Comment:
This library is a plugin of videojs, not videojs itself. So I think we need to remove homepage field, or find the homepage of this library.